modules/appeals_api/app/services/appeals_api/events/appeal_received.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  module Events
+    class AppealReceived
+      def initialize(opts)
+        @opts = opts
+        raise InvalidKeys unless required_keys?
+      end
+
+      def hlr_received
+        return unless Flipper.enabled?(:decision_review_hlr_email)
```

Comment:
Did we already add this Flipper setting in another PR? Did we make sure it was dependent on the global "send emails" flag as well (assuming it exists)?